content/vendor/snapshots/schedule.md

```diff
@@ -0,0 +1,13 @@
+---
+date: 2019-01-23
+linktitle: "Schedules"
+title: Schedules
+---
+
+Navigate to the `/app/<slug>/snapshots/schedule` page to configure automatic snapshots and to configure snapshot retention.
+
```

Comment:
If this is an admin console thing, I'd place this in the Admin Console section (see comment above). If this is a vendor thing, I'd probably combine with the "snapshots" page below and put it under packaging. I think the vendor side of this could be described in a single page.